command-line-flags-for-tidb-configuration.md

```diff
@@ -112,6 +112,11 @@ When you start the TiDB cluster, you can use command-line options or environment
 - Default: `"/tmp/tidb"`
 - You can use `tidb-server --store=unistore --path=""` to enable a pure in-memory TiDB.
 
+## `--proxy-protocol-fallbackable`
+
+- Controls whether to enable PROXY protocol fallback mode. When set to `true`, TiDB accepts client connections that do not follow the PROXY protocol or do not send a PROXY protocol header. By default, TiDB only accepts client connections that send a PROXY protocol header.
```

Comment:
```suggestion
- Controls whether to enable PROXY protocol fallback mode. When this parameter is set to `true`, TiDB accepts PROXY client connections and client connections without any PROXY protocol header. By default, TiDB only accepts client connections that send a PROXY protocol header.
```
It should be clarified whether the protocol header is sent by the client connection.